lib/task-data/tasks/dell-wsman-configure-bios.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017, Dell, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: "Dell Wsman Configure Bios",
+    injectableName: "Task.Dell.Wsman.ConfigureBios",
+    implementsTask: "Task.Base.Dell.Wsman.ConfigureBios",
+    options: {
+        attributes: null,
+        biosBootSequenceOrder: null,
+        hddSequenceOrder: null,
+        enableBootDevices: null,
+        disableBootDevices: null,
+        rebootJobType: null,
+        scheduledStartTime: null,
+        untilTime: null
```

Comment:
Create a schema for these options, then RackHD can use it to do upfront parameter validation and automatically generate a document for this task, this is a link about Options Schemas: http://rackhd.readthedocs.io/en/latest/rackhd/tasks.html?highlight=Task%20Schema#options-schema